python/packages/autogen-core/docs/src/user-guide/core-user-guide/cookbook/distributed-group-chat.ipynb

```diff
@@ -0,0 +1,258 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Distributed Round Robin Group Chat\n",
+    "This notebook demonstrates how to implement a distributed round-robin group chat. In this example we will create three {py:class} `WorkerAgentRuntime` runtime object, two chat participant and a group chat manager, and each of them will connect to an instance of  `WorkerAgentRuntimeHost` which implements the required grpc server.\n",
+    "You would only need to provide `OPENAI_API_KEY` variable below to be able to run this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "OPENAI_API_KEY = os.environ[\"OPENAI_API_KEY\"]"
```

Comment:
Let's add azure openai by following this: https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/framework/model-clients.html#azure-openai.

Reply:
Updated to Azure OpenAI supporting both AAD and API Token